python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -5046,8 +5180,21 @@ def test_rowwise_ops_datetime_dtypes(data, op, skipna):
 
     pdf = gdf.to_pandas()
 
-    got = getattr(gdf, op)(axis=1, skipna=skipna)
-    expected = getattr(pdf, op)(axis=1, skipna=skipna)
+    # TODO: This behavior seems erroneous in pandas. Why is the min/max over
```

Comment:
Are you referring to this behavior?

```
In [2]: df = pd.DataFrame([[1, pd.Timestamp.now()]])

In [3]: df.max(axis=1)
<ipython-input-3-6ecb52e231e7>:1: FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
  df.max(axis=1)
Out[3]:
0    1
dtype: int64

In [4]: pd.__version__
Out[4]: '1.5.2'
```

This will actually raise in 2.0

```
In [4]: df = pd.DataFrame([[1, pd.Timestamp.now()]])

In [5]: df
Out[5]:
   0                          1
0  1 2022-12-19 09:48:39.589243

In [6]: df.max(axis=1)
TypeError: '>=' not supported between instances of 'int' and 'Timestamp'

In [7]: pd.__version__
Out[7]: '2.0.0.dev0+929.g79b131e775'
```

Reply:
Hmm unfortunately I've forgotten exactly what confused me here. Reading my comment suggests that it wasn't _just_ the `numeric_only` behavior change (which the comment does address), but I can't reproduce anything else. Maybe I just confused myself into thinking there was more happening here than a simple dropping of non-numeric columns.